### PR TITLE
Update deploying multi-stage RecSys PoC nb

### PR DESCRIPTION
**Pull request by rnyak**

### CODE REVIEW

File: examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb [hunk 1]
```diff
@@ -25,10 +25,14 @@
   },
```
Comment (bschifferer) [inline examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:25]: <p>Gabriel renamed the notebook to <a href="https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb" target="_blank">02-Merlin-Models-and-NVTabular-integration.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/186/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

**Review by bschifferer — CHANGES_REQUESTED** [review]
We need to rename the notebook in the 01-Building-... notebook to [02-Merlin-Models-and-NVTabular-integration.ipynb](https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb)

File: examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb (hunk 1 above)
Comment (karlhigley) [reply examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:25]: @bschifferer These two notebooks don't look the same; I think they're different things.

Comment (bschifferer) [reply examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:25]: In [01-Building-Recommender-Systems-with-Merlin.ipynb](https://github.com/NVIDIA-Merlin/Merlin/pull/186/files/bc57ba599c84a4e3abc1112915a9756d4e005327#diff-c78bb349400fa13bcfb9de5457e74bd4648556f3e0a81621506f5c0c6648025b) we reference a notebook in Merlin Models ( [02-Merlin-Models-and-NVTabular-integration.ipynb](https://github.com/NVIDIA-Merlin/models/blob/main/examples/02-Merlin-Models-and-NVTabular-integration.ipynb) ).01-Building-Recommender-Systems-with-Merlin.ipynb uses the old filename and old url. We need to update the reference to the new filename/url

Comment (rnyak) [reply examples/Deploying-multi-stage-RecSys/01-Building-Recommender-Systems-with-Merlin.ipynb:25]: done.

**Review by bschifferer — APPROVED** [review]

**Pull request merged**